charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -49,7 +49,6 @@ data:
     query.max-memory={{ .Values.server.config.query.maxMemory }}
     query.max-memory-per-node={{ .Values.server.config.query.maxMemoryPerNode }}
     memory.heap-headroom-per-node={{ .Values.server.config.memory.heapHeadroomPerNode }}
-    discovery-server.enabled=true
```

Comment:
Is this removed now? i.e. specifying it causes an error?

Reply:
for what I have seen it just log a deprecation warning.